src/client/components/pages/Courses/__tests__/InstructorModal.test.tsx

```diff
@@ -0,0 +1,232 @@
+import React from 'react';
+import { render } from 'test-utils';
+import { TERM } from 'common/constants';
+import { RenderResult, within } from '@testing-library/react';
+import * as dummy from 'testData';
+import CourseInstanceResponseDTO from 'common/dto/courses/CourseInstanceResponse';
+import { stub } from 'sinon';
+import { strictEqual, deepStrictEqual, notStrictEqual } from 'assert';
+import InstructorModal from '../InstructorModal';
+
+describe('InstructorModal', function () {
+  let renderResult: RenderResult;
+  let testCourse: CourseInstanceResponseDTO;
+  let instructorNames: string[];
+  const term = TERM.FALL;
+  const { calendarYear } = dummy.cs50CourseInstance.fall;
+  const closeStub = stub();
+  const saveStub = stub();
+  describe('Rendering Instructor list', function () {
+    context('With No Instructors', function () {
+      beforeEach(function () {
+        testCourse = {
+          ...dummy.cs50CourseInstance,
+          fall: {
+            ...dummy.cs50CourseInstance.fall,
+            instructors: [],
+          },
+        };
+        renderResult = render(
+          <InstructorModal
+            isVisible
+            currentCourse={testCourse}
+            currentSemester={{ term, calendarYear }}
+            closeModal={closeStub}
+            onSave={saveStub}
+          />
+        );
+      });
+      it('Should not render any instructors', function () {
+        const instructors = renderResult.queryAllByRole('listitem');
+        strictEqual(instructors.length, 0);
+      });
+    });
+    context('With one instructor', function () {
+      beforeEach(function () {
+        testCourse = {
+          ...dummy.cs50CourseInstance,
+          fall: {
+            ...dummy.cs50CourseInstance.fall,
+            instructors: [
+              dummy.cs50CourseInstance.fall.instructors[0],
+            ],
+          },
+        };
+        instructorNames = testCourse.fall.instructors
+          .map(({ displayName }) => displayName);
+        renderResult = render(
+          <InstructorModal
+            isVisible
+            currentCourse={testCourse}
+            currentSemester={{ term, calendarYear }}
+            closeModal={closeStub}
+            onSave={saveStub}
+          />
+        );
+      });
+      it("Should render the instructor's display name", function () {
+        const instructors = renderResult.queryAllByRole('listitem');
+        strictEqual(instructors.length, 1);
+        const renderedNames = instructors
+          .map(({ textContent }) => textContent);
+        deepStrictEqual(renderedNames, instructorNames);
+      });
+      it('should render a delete button', function () {
+        const deleteButton = renderResult.queryByLabelText(
+          `Remove ${instructorNames[0]} from ${testCourse.catalogNumber}`,
+          { exact: false }
+        );
+        notStrictEqual(deleteButton, null);
+      });
+      it('Should not render any arrow buttons', function () {
```

Comment:
It looks like it does render arrow buttons that are just have no click action/`VARIANT.DEFAULT` colored.

Reply:
yeah, those should be cleared. Changed the test descriptions in